CONTRIBUTING.md

```diff
@@ -47,6 +47,7 @@ Here's a good workflow:
 - Outline your planned implementation. If you want help getting started, ask!
 - Follow the steps outlined below to develop locally. Once you have opened a PR, one of the `dbt-core` maintainers will work with you to review your code.
 - Add a test! Tests are crucial for both fixes and new features alike. We want to make sure that code works as intended, and that it avoids any bugs  previously encountered. Currently, the best resource for understanding `dbt-core`'s [unit](test/unit) and [integration](test/integration) tests is the tests themselves. One of the maintainers can help by pointing out relevant examples.
+- Check your formatting and linting with [Flake8](https://flake8.pycqa.org/en/latest/#), [Black](https://github.com/psf/black), and the rest of the hooks we have in our [pre-commit](https://pre-commit.com/) [config](https://github.com/dbt-labs/dbt-core/blob/75201be9db1cb2c6c01fa7e71a314f5e5beb060a/.pre-commit-config.yaml).
```

Comment:
Installing the pre-commit hooks is already covered in our internal docs since it's incorporated into the `Makefile`.  We don't generally provide development env. instructions for public consumption (we don't have docs on how to run tox, bumpversion, etc..) so it feels out of scope to provide them here.